configure.ac

```diff
@@ -92,6 +92,13 @@ if test "$enable_polkit" != "no"; then
 fi
 if test "$enable_ssh" != "no"; then
   PKG_CHECK_MODULES(libssh, [libssh >= 0.8.5])
+  if test "$pkg_failed" != "yes"; then
+    old_CFLAGS=$CFLAGS; CFLAGS=$libssh_CFLAGS
+    old_LIBS=$LIBS; LIBS=$libssh_LIBS
```

Comment:
Ugh @ CFLAGS/LIBS hack -- isn't this a job for `AC_CHECK_HEADER`?

Reply:
Hmm, AC_CHECK_HEADER(S) is meant to check whether a header exists and can be included, not whether it declares a certain function.  I think my "hack" is actually the best we can do.  That's a bit disappointing, but all we could ask for is a ready-made macro that does the exact same thing behind the scenes, I guess.  (Or we could check versions, but I'd rather check for the presence of the function.)

Reply:
Hmm, rather stunning that such a common case wouldn't exist already.. but okay. `AC_CHECK_LIB` could work as well.